sys/kern/sys_process.c

```diff
@@ -675,10 +681,12 @@ sys_ptrace(struct thread *td, struct ptrace_args *uap)
 	AUDIT_ARG_PID(uap->pid);
 	AUDIT_ARG_CMD(uap->req);
 	AUDIT_ARG_VALUE(uap->data);
-	addr = &r;
+
+	(void)c;
```

Comment:
???

Reply:
ok, this is due to a copy-paste from cheriabi_ptrace that I forgot to remove. I had the same reaction